### ISSUE TITLE
Winetricks fails to install PhysX

### ISSUE THREAD

**Issue by iQiexie** [Winetricks/winetricks#2210]
OS: Nobara 39

logs:

```
user@pc:~$ winetricks physx
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20240105-next - sha256sum: 39ad27b5de0fbc7878301389bab814cf172f7fe2837109a343771301a3134b32 with wine-9.5 (Staging) and WINEARCH=win64
Executing w_do_call physx
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_physx 
------------------------------------------------------
warning: /home/user/.wine/dosdevices/c:/Program Files (x86)/NVIDIA Corporation/PhysX/Engine/86C5F4F22ECD/APEX_Particles_x64.dll is not a regular file, not checking sha256sum
------------------------------------------------------
grep: warning: stray \ before /
Executing cd /home/user/.cache/winetricks/physx
Executing wine PhysX_9.21.0713_SystemSoftware.exe
0254:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
0254:fixme:exec:SHELL_execute flags ignored: 0x00000100
------------------------------------------------------
warning: Note: command wine PhysX_9.21.0713_SystemSoftware.exe returned status 9. Aborting.
------------------------------------------------------
```

32 bit wine version also doesn't help

```
user@pc:~$ mv .wine/ .wine.old
user@pc:~$ WINEARCH=win32 winetricks physx
------------------------------------------------------
Creating WINEPREFIX "/home/user/.wine" with WINEARCH=win32
------------------------------------------------------
wine: created the configuration directory '/home/user/.wine'
002c:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
002c:fixme:winediag:loader_init wine-staging 9.5 is a testing version containing experimental patches.
002c:fixme:winediag:loader_init Please mention your exact version when filing bug reports on winehq.org.
004c:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
0054:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
004c:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
004c:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
004c:err:ole:apartment_get_local_server_stream Failed: 0x80004002
004c:err:ole:start_rpcss Failed to open RpcSs service
0054:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hr 0x80004002
0054:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, hr 0x80004002
0054:err:ole:apartment_get_local_server_stream Failed: 0x80004002
007c:err:ntoskrnl:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
007c:err:ntoskrnl:ServiceMain Failed to load L"C:\\windows\\system32\\drivers\\dxgkrnl.sys"
007c:err:ntoskrnl:ServiceMain Failed to load L"C:\\windows\\system32\\drivers\\dxgmms1.sys"
009c:fixme:file:NtLockFile I/O completion on lock not implemented yet
009c:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00a4:fixme:file:NtLockFile I/O completion on lock not implemented yet
00a4:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00a4:fixme:msi:internal_ui_handler internal UI not implemented for message 0x0b000000 (UI level = 5)
00a4:fixme:msi:internal_ui_handler internal UI not implemented for message 0x0b000000 (UI level = 5)
009c:fixme:msi:internal_ui_handler internal UI not implemented for message 0x0b000000 (UI level = 1)
009c:fixme:msi:internal_ui_handler internal UI not implemented for message 0x0b000000 (UI level = 1)
wine: configuration in L"/home/user/.wine" has been updated.
0114:fixme:wineusb:query_id Unhandled ID query type 0x5.
0114:fixme:wineusb:query_id Unhandled ID query type 0x5.
0114:fixme:wineusb:query_id Unhandled ID query type 0x5.
0114:fixme:wineusb:query_id Unhandled ID query type 0x5.
Using winetricks 20240105-next - sha256sum: 39ad27b5de0fbc7878301389bab814cf172f7fe2837109a343771301a3134b32 with wine-9.5 (Staging) and WINEARCH=win32
Executing w_do_call physx
Executing load_physx 
------------------------------------------------------
warning: /home/user/.wine/dosdevices/c:/Program Files/NVIDIA Corporation/PhysX/Engine/86C5F4F22ECD/APEX_Particles_x64.dll is not a regular file, not checking sha256sum
------------------------------------------------------
grep: warning: stray \ before /
Executing cd /home/user/.cache/winetricks/physx
Executing wine PhysX_9.21.0713_SystemSoftware.exe
0220:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
0220:fixme:exec:SHELL_execute flags ignored: 0x00000100
------------------------------------------------------
warning: Note: command wine PhysX_9.21.0713_SystemSoftware.exe returned status 9. Aborting.
------------------------------------------------------
```

**Comment by SveSop** [comment]
Each fortnight i await a random fix for this.. But wine-staging-9.7 same issue still.

**Comment by SveSop** [comment]
@iQiexie 
In case you are interested in compiling wine yourself, reverting this patch seems to fix installing PhysX either manually or using winetricks:

https://github.com/wine-mirror/wine/commit/85d029e3b01f6dd35a86cc07796af982d66e4a03

I might post a bug about this on WineHQ, but chances are "they" are going to blame nvidia for some crap and end up with yet another "WontFix".. But hey...

**Comment by austin987** [comment]
https://bugs.winehq.org/show_bug.cgi?id=56606

**Comment by mirh** [comment]
Fixed in 9.10
